### DISCUSSION TITLE
Add way to define a workflow_dispatch input to choose from values

### DISCUSSION THREAD

**Discussion by anatawa12** [community/community#8588] [Actions] [answered]
Currently, all inputs of workflow_dispatch are any string. 
However, we may want to limit the string to a few values. 
Furthermore, For such a value, I want to choose a value from the dropdown/pulldown menu.
Could you add a feature to do this?
In my opinion, `on.workflow_dispatch.inputs.<input>.values` attribute is a good place to define the list of expected value.

**Searched terms:**
I searched with the following words on this category:
- dropdown
- drop down
- pulldown
- pull down
- one of values
- choose
- workflow_dispatch
- enum

**Comment by ethomson** [comment, marked as answer]
Hi @anatawa12 - I think that you're looking for the `choice` type for workflow dispatch events that [we announced last month](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/).

**Reply by anatawa12** [reply]
Thank you! I forgot to check the documentation for English.

**Reply by ethomson** [reply]
I think that we can improve the docs here a bit to help discoverability.  Thanks for your question!

**Reply by igorcosta** [reply]
Would be cool if we had a dedicated session in the docs for the types of inputs @ethomson

**Reply by silkfire** [reply]
@ethomson Agree, the current documentation doesn't explain what the different `type` properties actually do: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs

**Reply by patriziobrunops** [reply]
has the doc been udpated to describe the different input types?
